revolutionpi/analog_pin.go

```diff
@@ -35,3 +41,53 @@ func (pin *analogPin) Close(ctx context.Context) error {
 	// There is nothing to close with respect to individual analog _reader_ pins
 	return nil
 }
+
+func (b *revolutionPiBoard) WriteAnalog(ctx context.Context, pin string, value int32, extra map[string]interface{}) error {
```

Comment:
moved this to the analog file to keep most of the analog related code in the same place